structuredheadings/plugin.js

```diff
@@ -173,6 +183,9 @@
       ]
     };
 
+    editor.config.autonumberDefaultScheme =
+    editor.config.autonumberDefaultScheme || "1.1.1.1.1.";
```

Comment:
can this be more inner indented to make it obvious is the value of the above assignment?